### ISSUE TITLE
Scenario name as foreign key

### ISSUE THREAD

**Issue by EvaWie** [openego/eGon-data#177]
We liked the idea of defining the scenario name, so that it can be used as foreign key.

Which columns should/could the scenario table have?
- scenario name
- scenario description (ca. 3 sentences)
- scenario year
- weather year
- maybe heat sector assumptions: heat demand reduction factors and district heating share (for Germany)
- ...

Who will implement this table?

**Comment by gplssm** [comment]
Right now, assumptions are stored decentrally (i.e. district heating share, parameters for NEP). For scenario variations it might be helpful to have some assumptions and parameters stored centrally. This might be relevant for demand data of newly introduced sectors in particular. So please, if you already know that you're data has assumptions that might be interesting for a scenario variation, please add it here. By doing this we get a better overview about this scenario table should look like.

@nesnoj for example the number of BEVs might be a valuable scenario variation, right?

**Comment by nesnoj** [comment]
> @nesnoj for example the number of BEVs might be a valuable scenario variation, right?

Yes, and other params too.
To avoid bloating the columns it may be a good option to store it as [json](https://www.postgresql.org/docs/current/datatype-json.html).

**Comment by ClaraBuettner** [comment]
I added a first draft of the scenario-table on [this branch](https://github.com/openego/eGon-data/tree/features/%23177-create-scenario-table).
I wasn't sure in which schema I should add it (it is a combination of society, demand, supply... data) and preliminary added it to the grid schema (because it will contain information about all datasets and sectors). 

The parameters of each sector and scenario should be written in [the corresponding dict of parameters.py](https://github.com/openego/eGon-data/blob/features/%23177-create-scenario-table/src/egon/data/importing/scenarios/parameters.py). If the list of parameters will be too long, we can think of creating separate files for each sector.
I added some example parameters and tested them in the demandregio import  46004231af5600b8cda27eab82776c0c3624bf19.
